### PR TITLE
[CLB] Implement Mazzy, Truesword Paladin

### PR DESCRIPTION
**Pull request by PurpleCrowbar**
Implements Mazzy, Truesword Paladin from CLB

### CODE REVIEW

**Comment by PurpleCrowbar** [comment]
[[Mazzy, Truesword Paladin]]

File: Mage.Sets/src/mage/cards/m/MazzyTrueswordPaladin.java [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{1}{R}{G}{W}");
+
+        this.addSuperType(SuperType.LEGENDARY);
+        this.subtype.add(SubType.HALFLING);
+        this.subtype.add(SubType.KNIGHT);
+        this.power = new MageInt(3);
+        this.toughness = new MageInt(4);
+
+        // Whenever an enchanted creature attacks one of your opponents, it gets +2/+0 and gains trample until end of turn.
+        this.addAbility(new MazzyAttackTriggeredAbility());
```
Comment (Alex-Vasile) [inline Mage.Sets/src/mage/cards/m/MazzyTrueswordPaladin.java:51]: You should make use of `AttacksAttachedTriggeredAbility`

Comment (Alex-Vasile) [reply Mage.Sets/src/mage/cards/m/MazzyTrueswordPaladin.java:51]: nvm

File: Mage.Sets/src/mage/cards/m/MazzyTrueswordPaladin.java [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+        if (super.checkTrigger(event, game)) {
+            Player defender = game.getPlayer(event.getTargetId());
+            if (defender == null) {
+                return false;
+            }
+            Set<UUID> opponents = game.getOpponents(this.getControllerId());
+            if (opponents != null && opponents.contains(defender.getId())) {
+                for (Effect effect : this.getEffects()) {
+                    effect.setTargetPointer(new FixedTarget(event.getSourceId(), game));
+                }
```
Comment (Alex-Vasile) [inline Mage.Sets/src/mage/cards/m/MazzyTrueswordPaladin.java:97]: Could use `getEffects().setTargetPointer(new FixedTarget(attachedPermanent, game));` instead

**Review by Alex-Vasile — APPROVED** [review]

File: Mage.Sets/src/mage/cards/m/MazzyTrueswordPaladin.java (hunk 2 above)
Comment (PurpleCrowbar) [reply Mage.Sets/src/mage/cards/m/MazzyTrueswordPaladin.java]: Done

**Pull request merged**